### ISSUE TITLE
Carthage not found warning on trying to launch the app on the simulator 

### ISSUE THREAD

**Issue by Rnij** [appium/appium#9305]
## The problem

I am trying to launch the app on a simulator. It gives a warning Carthage not found when Carthage is already installed.

## Environment

Appium version: 1.7.0 
* Desktop OS/version used to run Appium: Mac OS Sierra 10.12.5
Node.js version (unless using Appium.app|exe):
Mobile platform/version under test:10.3
Real device or emulator/simulator:iPad Pro Simulator
Appium CLI or Appium.app|exe: 1.2.1

## Details

Following are the Desired Capabilities that I am using

{
"platformName": "iOS",
"platformVersion": "10.3.3",
"deviceName": "iPad Pro",
"udid": "21dbb46b868acba3a5bc53832e2b571f9611b547",
"app":"/Users/rgupte/Desktop/POD.app",
"automationName": "XCUITest",
"xcodeOrgId": "AUP79D422L",
"xCodeSigningId": "iphone Developer"
}
When I start the session on Appium it displays that Carthage was not found when I have already installed carthage. If I run the command 
which Carthage it displays the following which is also included in PATH.
/usr/local/bin/carthage 

On giving echo $PATH it displays 
$/usr/local/bin/carthage:$/usr/local/Cellar/ideviceinstaller/HEAD-442f670_3/bin:$/usr/local/Cellar/libimobiledevice/1.2.0_2/bin:/Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home/bin:$/Library/Java/JavaVirtualMachines/jdk1.8.0_141.jdk/Contents/Home/bin:$/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin

## Link to Appium logs

[logs.txt](https://github.com/appium/appium/files/1330320/logs.txt)

**Comment by imurchie** [comment]
@dpgraham Seems like more problems with replicating the environment.

**Comment by mykola-mokhnach** [comment]
The PATH you see in terminal might not be the same, which is passed to Appium process. Check the output of `ps -wwE -p <appium_pid>` command to make sure `/usr/local/bin` is there.

**Comment by Rnij** [comment]
I checked out the output of the command and it does not list /usr/local/bin. See below. Can you please advise how Appium can use the path listed on terminal.

ps -wwE -p 8580
  PID TTY           TIME CMD
 8580 ??         0:22.69 /Applications/Appium.app/Contents/MacOS/Appium TMPDIR=/var/folders/d4/jvmfnng92x12hpg4x82wslzsmk7fjt/T/ __CF_USER_TEXT_ENCODING=0x6923BA3A:0x0:0x0 SHELL=/usr/bin/ksh HOME=/Users/rgupte Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.cQwQBsxzpI/Render SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.vMfrpJamnr/Listeners PATH=/usr/bin:/bin:/usr/sbin:/sbin LOGNAME=rgupte XPC_SERVICE_NAME=io.appium.desktop.12204 USER=rgupte XPC_FLAGS=0x1

**Comment by mykola-mokhnach** [comment]
I don't know - this depends on the way your appium process is executed. Usually each newly created child process automatically inherits the environment from its parent parent, but one can override it by using special startup flags and configure child process environment on his own.
Closed as duplicate of https://github.com/appium/appium/issues/9265

**Comment by shabsi4u** [comment]
Following up from @Rnij 's comment, I am able to see the /usr/local/bin in the Appium process path, and in the /etc/paths file. I can't use the solution in #9265 since I am trying to use the Appium Mac app with simulator. Could you please help unblock me on this?
